src/VocabGamePlay.scss

```diff
@@ -69,7 +53,7 @@
     .card-correct {
       animation: correct-animation ease-out 2s forwards;
       &:hover{
-        transform: scale(1)
+        transform: scale(1.1);
       }
     }
```

Comment:
after playing with the game a little, i think the correct/incorrect animations should happen a tad faster

Reply:
cards with disabled styling should also not transform scale onHover.

Reply:
I thought you might want to address the hover-scaling thing because I saw a commit from you from last time that said "card scales only when selectable"? So I thought maybe you already knew what was up with that... I can take a look at that if you want instead.

You're right on the animation speed, I'll change that next go-around.